apps/docs/content/docs/collaboration.mdx

```diff
@@ -0,0 +1,133 @@
+
+### Synchronizing data
+
+For more information about how to synchronize the tldraw editor's store with other processes (i.e. how to get data out and put data in, including from remote sources) see the [Persistence](/docs/persistence) page.
+
+### User assets
+
+As well as synchronizing the rapidly-changing document data, tldraw also needs a way to store and retrieve large binary assets like images or videos. You'll need to make sure your backend can handle asset uploads and downloads, and implement a service for turning files and links into assets that the editor can use.
+
+You can use the [Editor](?)'s asynchronous external asset handlers to generate assets from content that a user drops, pastes, or adds to the canvas. For example, you might upload images to a storage bucket and return an asset that includes their source URL; or pull favicons and social images in order to create bookmarks from pasted URLs.
+
+```tsx
+<Tldraw
+	store={store}
+	onMount={(editor) => {
+		editor.registerExternalAssetHandler('file', createAssetFromFile)
+		editor.registerExternalAssetHandler('url', createAssetFromUrl)
+	}}
+/>
+```
+
+Read more about [how assets work in tldraw](/docs/assets). You can also refer to our simple example [client](https://github.com/tldraw/tldraw/blob/main/apps/simple-server-example/src/client/App.tsx) and [server](https://github.com/tldraw/tldraw/blob/main/apps/simple-server-example/src/server/unfurl.ts) code for examples of asset handling.
+
+### User presence
+
+The tldraw SDK has support for displaying the 'presence' of other users. Presence information consists of the user id, name, and color, as well as their canvas context, such as their current page, pointer position, selection, viewport.
+
+A user's presence is stored as `instance_presence` ([TLInstancePresence](?))records in the editor's store, however the data itself may be derived from other sources.
+
+We provide a helper, [createPresenceStateDerivation](?), for constructing a reactive signal for an `instance_presence` record locally, which can then be sent to other clients somehow.
+
+To the editor, any `instance_presence` records that belong to other users (i.e. those with a different user id than the editor's configured user id) are considered "collaborators". The editor has two methods, `getCollaborators` and `getCollaboratorsOnCurrentPage`, which return reactive signals about the current collaborators.
+
+See the [yjs-example](https://github.com/tldraw/tldraw-yjs-example/blob/45dfe7589ae44e4aaf9bb7359d0372f3a88bd597/src/useYjsStore.ts) for an example of how these records may be created, shared, and updated.
+
+### Collaboration UI
+
+The `tldraw` library includes several components specifically designed for collaboration.
+
+- Collaborator cursors: [`TLComponents#CollaboratorCursor`](?)
+- Cursor chat: [`TLComponents#CursorChatBubble`](?)
+- Offline indicator: [`TLComponents#TopPanel`](?)
+- Collaborator list: [`TLComponents#SharePanel`](?)
```

Comment:
If you click these links, you'll see they don't go anywhere super useful - they just sort of list out the components you'd override to customize them. In my original version of this, I made sure to explain that the components prop is how these should be overridden before i listed these out. I think they're just sort of confusing otherwise. Can we move this below the line about TLComponents?